tapdance/conjure.go

```diff
@@ -59,9 +59,13 @@ func DialConjure(ctx context.Context, cjSession *ConjureSession) (net.Conn, erro
 	Logger().Debugf("%v Successfully sent registrations, sleeping for: %v ms", cjSession.IDString(), toSleep)
 	time.Sleep(toSleep)
 
-	Logger().Debugf("%v Woke from sleep, attempting to Connect ...", cjSession.IDString())
-	return registration.Connect(ctx)
-	// return Connect(cjSession)
+	Logger().Tracef("%v Woke from sleep, attempting to Connect ...", cjSession.IDString())
+
+	conn, err := registration.Connect(ctx)
+	if conn != nil && err == nil {
+		go StatsReporting(cjSession.stats)
```

Comment:
This is on me, but both `cjSession`, and `registration` track stats and I'm not sure exactly which of the stats we want to send via this channel. I think we probably want `registration.stats`. This is mostly an issue of deciding exactly what we want to track and send over this channel. 

It may be worth opening a different issue to deal with this, as this change does complete the task of setting up the channel to communicate stats which is the issue that we opened here. 

thoughts @nzimm?

Reply:
I think reporting the correct stats protobuf probably falls under the scope of this ticket.
I'm just not sure what is tracked differently between `cjSession.stats` and `registration.stats`